packages/ui/src/components/banner-container/banner-container.tsx

```diff
@@ -1,9 +1,11 @@
 import {Component, h} from 'preact';
 import {ContribLogger, getContribLogger} from '@playkit-js-contrib/common';
 import * as styles from './_banner-container.scss';
+import BannerConfig = KalturaPlayerContribTypes.BannerConfig;
```

Comment:
i don't think this import is needed, can you check?

Reply:
done